x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/55_search_application_search.yml

```diff
@@ -45,6 +45,14 @@ setup:
               params:
                 field_name: field1
                 field_value: value1
+            dictionary:
```

Comment:
`dictionary` now allows the full JSON Schema syntax, including `required` parameters and limiting `additionalProperties`.

I think we should change the name from `dictionary` to `param_schema`. It is more descriptive and in line with using a JSON schema for validation.

Reply:
I like your rename suggestion 👍